File: Atomicy.Domain/Entities/Reservation.cs
using Atomicy.Domain.Common;
using System;

namespace Atomicy.Domain.Entities
{
    public class Reservation : AuditableEntity
    {
        public Guid ReservationId { get; set; }
        public Guid DemandId { get; set; }
        public Guid UserId { get; set; }
        public Guid FirmId { get; set; }
        public string Note { get; set; }
        public DateTime ReservationDate { get; set; }
        public bool Completed { get; set; }
        public DateTime CarryDate { get; set; }
        public Demand Demand { get; set; }
        public Firm Firm { get; set; }

    }
}

File: Atomicy.SPA/src/core/models/reservation.ts
import { Guid } from "guid-typescript";

export class Reservation {
  reservationId!: Guid;
  demandId!: Guid;
  userId!: Guid;
  firmId!: Guid;
  note!:string;
  reservationDate!: Date;
  carryDate!: Date;
  completed!: boolean;
}
